### PR TITLE
Add simple server to project

### PR DESCRIPTION
**Pull request by mariamera**
## Review checklist

- [x] Add the corresponding label
- [x] Would you be proud of this work? 

## Testing

Steps to test the change:
1. Download Branch
2. Run `npm start` 
3. Browse http://127.0.0.1:8081/ 
4. see if index.html is working correctly 
## Deploy checklist

- [ ] Rebase and squash your branch so it is one commit ahead of `origin/master`
- [ ] Use your PR title and the entire summary above the line as the commit message
- [ ] Delete branch after merging

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+var express = require('express');
+var app = express();
+app.use(express.static('public'));
+
+app.get('./public/index.html', function (req, res) {
+  res.sendFile(__dirname + '/' + 'index.html');
+});
+
+app.get('*', function (req, res) {
+  res.send('<h1> PAGE NOT FOUND </h1>', 404);
```
Comment (fxisco) [inline server.js]: *Magic Number*: 404. Use constants instead

File: server.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+});
+
+var server = app.listen(8081, function () {
```
Comment (fxisco) [inline server.js]: **Magic Number**: 8081

File: server.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  var host = server.address().address;
+  var port = server.address().port;
+
+  console.log('Example app listening at http://%s:%s', host, port);
```
Comment (fxisco) [inline server.js]: Are you able to use backticks?

File: server.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+var express = require('express');
+var app = express();
+app.use(express.static('public'));
+
+app.get('./public/index.html', function (req, res) {
```
Comment (fxisco) [inline server.js]: Jumm 🤔 why not have this as `/`

File: server.js (hunk 3 above)
Comment (mariamera) [reply server.js]: nop , eslint does not allow me : 
` error  Strings must use singlequote  quotes` 👍

File: server.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const NOT_FOUND_ERROR = {
+  'message': '<h1> PAGE NOT FOUND !</h1>',
+  'error_type': 404 };
```
Comment (fxisco) [inline server.js]: why?  😱 `error_type` -> errorType or errorCode better

File: server.js [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+var express = require('express');
+var app = express();
+app.use(express.static('public'));
+
+app.get('/', function (req, res) {
+  res.sendFile(__dirname + '/' + 'index.html');
+});
+
+app.get('*', function (req, res) {
+  res.send(NOT_FOUND_ERROR['message'], NOT_FOUND_ERROR['error_type']);
```
Comment (fxisco) [inline server.js]: Avoid typos using dot notation

File: server.js [hunk 7]
```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const NOT_FOUND_ERROR = {
```
Comment (fxisco) [inline server.js]:
```
const HTTP_CODES = {
  NOT_FOUND: 404
};
```

File: server.js (hunk 3 above)
Comment (fxisco) [reply server.js]: Could you please check how could we allow them?

**Comment by MariaFdezJ** [comment]
I'm watching now that we have a `public` folder, which has a `src` folder, exactly as the one outside of `public`. Are both needed?  
image


Also, this is not related with this specific branch (saw it on master too), but taking note here that in my browser I get this two errors. Is there something I'm missing here? Thanks in advance!
image

**Comment by mariamera** [comment]
@MariaFdezJ Good Catch with the src folder duplicated ! Thanks, and about the `module is not define` it has to do with the `module.exports` not supported by the browsers.

File: server.js [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+  'NOT_FOUND': 404
+},
+PORT = 8081;
+
+var express = require('express');
+var app = express();
+app.use(express.static('public'));
+
+app.get('/', function (req, res) {
+  res.sendFile(__dirname + `/` + `index.html`);
```
Comment (fxisco) [inline server.js]: \`${__dirname}/index.html\`

**Pull request merged**